aries-site/src/components/cards/ContentCard.js

```diff
@@ -1,19 +1,28 @@
 import React, { forwardRef } from 'react';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { Box, Image, Text } from 'grommet';
 import { Identifier, Tile } from 'aries-core';
 import { PreviewImageCard } from './PreviewCard';
 import { useDarkMode } from '../../utils';
 
+const StyledTile = styled(Tile)`
+  transition: all 0.3s ease-in-out;
+  :focus,
+  :hover {
+    transform: scale(1.01, 1.01);
+  }
+`;
+
 export const ContentCard = forwardRef(({ topic, ...rest }, ref) => {
   const { description, name, parent, preview } = topic;
   const [isFocused, setIsFocused] = React.useState(false);
   const darkMode = useDarkMode();
   return (
-    <Tile
+    <StyledTile
       align="start"
       background="background-front"
-      elevation={isFocused ? 'medium' : 'none'}
+      elevation={isFocused ? 'medium' : 'xsmall'}
```

Comment:
What was the reasoning for adding default elevation? Can we leave it as flat by default?

Reply:
Intended to communicate that the card is sitting on top of the surface. The piece of paper laying on your desk has elevation, albeit slight. When you pick it up, the elevation increases. When you set it back down, it doesn't get absorbed into surface, rather it goes back to having slight elevation.

So, attempting to mimic that experience.

It looks nice too. ;)

Reply:
Also, FYI, working with Greg on a couple tweaks to how elevations are currently defined.